docs/application/tizen-studio/release-notes/5-5-release-notes.md

```diff
@@ -0,0 +1,72 @@
+# Tizen Studio 5.5 Release Notes
+
+- Release Date: Oct 31, 2023
+
+## IDE and tools
+
+### New features
+- Tizen Studio
+  - Added unified profile called tizen and removed mobile and wearable profiles from Tizen 8.0 version onwards.
+  - Added support for 64 bit emulator for tizen-8.0 profile.
+  - Tizen Studio and tools are now supported on AMD PCs.
+  - Added support for Tizen profile in Dynamic Analyzer.
+- Certificate Manager
+  - Added new distributor certificates from Tizen 8.0 version onwards.
+  - Added support for creation of new distributor certificate from certificate-manager UI.
+- Visual Studio (Windows)
+  - Added support for .NET diagnostics tools - dump, gcdump and trace for Tizen dotnet projects.
+  - Heaptrack configuration page has been added in Memory Profiler.
+- Visual Studio Code
+  - Added support for heaptrack profiler for Tizen dotnet projects.
+
+### Fixed bugs
+  
+  - Fixed the issue of Battery Historian not launching in Windows.
+  - Fixed the error in manifest editor for custom 8.0 profile.
+  - Fixed the issue of native apps not launching on shell enabled TV devices from VS plugin.
+  - Fixed dotnet app build failure in VS plugin when .NET SDK is installed in multiple locations.
+  - Handled on-demand package installation which contained version file similar to RPM packages.
+  - Fixed the issue of dotnet workload installation in Windows on VSCode plugin.
+
+### Known issues
+
+  - On macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
+  - In case you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+  - On macOS, from Catalina and above versions, Native Templates-5.5 will not build with CLI when the compiler is set to **GCC**.
```

Comment:
```suggestion
  - On macOS, since Catalina and above versions, Native Templates-5.5 will not build with CLI when the compiler is set to **GCC**.
```